src/react-components/src/components/01-atoms/Images/Icons/Icon.tsx

```diff
@@ -0,0 +1,46 @@
+
+import * as React from "react";
+import Icons from "@nypl/design-system-icons/dist/img/sprite/sprite.svg";
+import bem from "../../../../utils/bem";
+
+  // Wrapper Class for Icon
+  export interface IconProps {
+    name: string;
+    blockName: string;
+    modifiers?: string[];
+    decorative: boolean;
+    role?: string;
+    title?: string;
+    desc?: string;
+  }
+
+  export default class Icon extends React.Component<IconProps, {}> {
```

Comment:
Not necessary for this PR, but look into functional components in React. This component is just a presentational component that doesn't have any logic, so writing it as a function can reduce some code (and provide more efficiency later on according to React).

Reply:
👍